### PR TITLE
Remove "gate" option completely

### PR DESCRIPTION
**Pull request by Rossmaxx**
Gate knob is one of the most broken "features" and per @LostRobotMusic, It shouldn't even exist.

### CODE REVIEW

File: data/presets/Monstro/Growl.xpf [hunk 1]
```diff
@@ -44,11 +44,11 @@
             <attribute name="plugin" value="Clip"/>
           </key>
         </effect>
-        <effect name="dualfilter" autoquit_denominator="4" on="1" autoquit_numerator="4" wet="1" autoquit_syncmode="0" gate="0.02" autoquit="1">
+        <effect name="dualfilter" autoquit_denominator="4" on="1" autoquit_numerator="4" wet="1" autoquit_syncmode="0" autoquit="1">
```
Comment (Rossmaxx) [inline data/presets/Monstro/Growl.xpf:47]: Should I do this for rest of the presets?

**Comment by bratpeki** [comment]
How do effects now look in the UI? Could you drop a screenshot?

**Comment by Rossmaxx** [comment]
I haven't compiled yet tbh. I'll update soon

**Comment by Rossmaxx** [comment]
image

@bratpeki here you go

**Review by LostRobotMusic — COMMENTED** [review]
In all of the Effects, you're removing the `checkGate` checks without also removing the unused `outSum` variables.

**Comment by Rossmaxx** [comment]
Good catch, I'll check

**Comment by messmerd** [comment]
Wouldn't it be easier if I just remove the gate in #7484? I'm already touching that code

**Comment by DomClark** [comment]
The way the "stop processing effects without input" feature turns off effects is by checking whether the output level falls below the gate value. Removing gate checking entirely will break this feature. The idea is to remove the gate knob and force a gate value of zero, not to remove all code relating to the gate.

**Comment by Rossmaxx** [comment]
I looked at @messmerd 's PR again and the diff is similar so it would be better to consolidate. 

Since I'm closing, I'm leaving dom's point to messmerd

**Comment by messmerd** [comment]
It looks like #7438 removes the Gate knob, so hopefully we can merge that soon and I won't have to do it in my PR.

**Pull request closed without merging**